abjad/tools/topleveltools/show.py

```diff
@@ -26,7 +26,7 @@ def show(expr, return_timing=False, **kwargs):
     '''
     from abjad.tools import systemtools
     from abjad.tools import topleveltools
-    assert '__illustrate__' in dir(expr)
+    assert hasattr(expr, "__illustrate__")
```

Comment:
Use single quotes instead of double, except when single quotes are part of the string in question.